www/apps/ui/src/content/docs/components/focus-modal.mdx

```diff
@@ -37,3 +37,37 @@ This component is based on the `div` element and supports all props of this elem
 ### FocusModal.Body
 
 This component is based on the `div` element and supports all props of this element.
+
+## Controlling Open State
+
+---
+
+If you want to control whether the Focus Modal is open, you have to pass the `FocusModal` component the following props:
+
+1. Set `defaultOpen` to `false` to ensure it's not open by default.
```

Comment:
Setting `defaultOpen` is not necessary, and does not make a lot of sense in the controlled case as we are setting open to the `open` returned by `useState`. It only has an effect if the modal is uncontrolled, and you for example want it to be open per default, rather than closed which is the default initial state of an uncontrolled modal.